CONTRIBUTING.md

```diff
@@ -238,3 +238,9 @@ By contributing to this repository, you agree to license your work under the
 MIT license unless specified otherwise in `contrib/debian/copyright` or at 
 the top of the file itself. Any work contributed where you are not the original 
 author must contain its license header with the original author(s) and source.
+
+If you do not know where the work comes from and/or its license terms, it may
+not be contributed until that is resolved. In particular, anything generated by
+AI or LLMs derived from undisclosed or otherwise non-MIT-compatible inputs
+(including, but not limited to, ChatGPT, GitHub Copilot, and Meta LLaMA) cannot
```

Comment:
I would recommend to drop any reference to a corporate entity, or one of its product in Bitcoin Core documentation, to avoid a mischaracterization of what they’re doing (whatever one personal opinion).

We already mentioned Github a lot in the contributing.md, though only as the technical platform where contributions are happening, not taking a stance on one of their product.

Reply:
I mentioned these specifically because:

1. ChatGPT is the most popularly known, and most likely to be searched for if someone is considering using it.
2. GitHub promotes use of Copilot heavily, and we are using GitHub.
3. Meta is falsely advertising LLaMA as open source, and many people are just believing that without verifying. (The source code is not available, and the license is not permissive)

Reply:
I think it's fine to mention these examples.

Reply:
I think a) there is no certainty ChatGPT / LLaMa will be the most popular framework 12 / 18 months from now and I don’t think we’re going to update contributing rules everytime and b) Meta is a registered trademark of a commercial entity and I think it’s better to not give the appearance Bitcoin Core the project is supportive or supported or linked to Meta in anyway.